src/golang/lib/workflow/operator/operator.go

```diff
@@ -129,7 +129,10 @@ func NewOperator(
 	var jobManager job.JobManager
 	var err error
 
-	if dagJobManager == nil {
+	if dagJobManager == nil && opEngineConfig.Type != shared.AirflowEngineType {
```

Comment:
As commented, we do not need a job manager for Airflow operators, since there is job being deployed